### ISSUE TITLE
Error: ModuleNotFoundError: No module named '__version'

### ISSUE THREAD

**Issue by apitsos** [noDRM/DeDRM_tools#212]
Hi there!

I downloaded the version 10.0.3 (latest) from your GitHub repository, and I tried to install the DeDRM_plugin in Calibre 6.8.0. I faced the following problems:

It's supposed that I should add the file "DeDRM_plugin.zip", according to your instruction, but this file was not existed in the repository of the version 10.0.3 I downloaded. But I realized that it should be just a zip file of the contents of the folder "DeDRM_plugin". So I created a zip myself. But when trying to Load the plugin, using the button "Load plugin from file", then I am getting the following errror:

calibre, version 6.8.0
ERROR: Unhandled exception: <b>ModuleNotFoundError</b>:No module named '__version'

calibre 6.8  embedded-python: True
Windows-10-10.0.19045-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19045')
Python 3.10.1
Windows: ('10', '10.0.19045', 'SP0', 'Multiprocessor Free')
Interface language: None
Successfully initialized third party plugins: Obok DeDRM (10, 0, 3)
Traceback (most recent call last):
  File "calibre\gui2\preferences\plugins.py", line 325, in add_plugin
  File "calibre\customize\ui.py", line 479, in add_plugin
  File "calibre\customize\ui.py", line 64, in load_plugin
  File "calibre\customize\zipplugin.py", line 307, in load
  File "importlib\__init__.py", line 126, in import_module
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "calibre\customize\zipplugin.py", line 206, in exec_module
  File "calibre_plugins.dedrm.__init__", line 99, in <module>
ModuleNotFoundError: No module named '__version'

In the meantime, in order to prove if I have done something wrong with the zip file, I followed the same procedure and created a zip file with the contents of the folder "Obok_plugin" and that file worked like a charm. I was able to load this particular plugin in Calibre 6.8.0 without any issue.

Would you be so kind to check it out and let me know what is wrong and how could I load the DeDRM plugin in Calibre?

Did I mentioned that I am doing that in Windows?

Thanks a lot in advance for your prompt attention to this matter!

**Comment by ElleKayEm** [comment]
Sounds like you don't have the correct file. 
 Download DeDRM_tools_10.0.3.zip (first link under Assets) from the release page: https://github.com/noDRM/DeDRM_tools/releases/tag/v10.0.3

Unzip that and load DeDRM_plugin.zip.

**Comment by ElleKayEm** [comment]
Just saw your comment about Obok.  Yes, it seems Obok will allow you to just zip up the source code folder yourself.  But with DeDRM, always use that first file under Assets on the release page where both plugins are already zipped up and ready to load.

**Comment by apitsos** [comment]
Hi again,

Thanks a lot for the information provided. I managed to install the plugin, but I am still having some issues. First of all I selected the install plugin from within the "Calibre" and I clicked the button "Customize plugin". Then under the vertical menu with the buttons I clicked the "Adobe Digital Editions ebooks" and then it imported my key from the Authorized Adobe Digital Editions application. I tried to save it then, but I am getting the following error:

calibre, version 6.8.0
ERROR: Unhandled exception: <b>Error</b>:decoding with 'hex' codec failed (Error: Non-hexadecimal digit found)

calibre 6.8  embedded-python: True
Windows-10-10.0.19045-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19045')
Python 3.10.1
Windows: ('10', '10.0.19045', 'SP0', 'Multiprocessor Free')
Interface language: None
Successfully initialized third party plugins: DeDRM (10, 0, 3) && Obok DeDRM (10, 0, 3)
Traceback (most recent call last):
  File "encodings\hex_codec.py", line 19, in hex_decode
binascii.Error: Non-hexadecimal digit found

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\angel\AppData\Roaming\calibre\plugins\DeDRM.zip\config.py", line 585, in export_key
binascii.Error: decoding with 'hex' codec failed (Error: Non-hexadecimal digit found)


And finally, I am not aware of how to export an unlocked eBook from Calibre. I want to export an EPUB file (actually it has the file extension .acsm), which will be used to reMarkable2.

Would you be so kind to help me with this issues?

**Comment by apitsos** [comment]
I also wanted to add the following info. After importing the eBook in Calibre, I am right-clicking on it and select the option "Save to disk" > "Save only EPUB format to disk". I select a folder and I click save. But then I am getting the below error message:

calibre, version 6.8.0
ERROR: Error while saving: Failed to save any books to disk, click "Show details" for more information

Failed to save: "xxx" by "yyy zzz" to disk, with error:
    Requested formats not available

I have removed the name of the book and the author, for obvious reasons.

**Comment by ElleKayEm** [comment]
The acsm file is not the ebook.  After opening the acsm file in ADE (authorized with your Adobe ID), add the downloaded epub (or pdf) to calibre.  Don't try to add your Adobe Key from within the plugin customization.  Just add the book file to calibre and let DeDRM automatically retrieve the key.  (ADE needs to be on the same machine and user as calibre/DeDRM.)  You couldn't save the epub format because you only had the acsm file in there.

**Comment by apitsos** [comment]
Hi again,

I would like to thank you very much for your help. The whole key in my case was that I shouldn't import my Adobe ID keys from within the plugin. As soon as I removed them and restarted the "calibre", it worked like a charm!

There is only one last question. I saw on a video that someone converted them EPUB into MOBI. I tried that as well and then I saved the EPUB file. It worked perfectly fine on a non-Authorized Kobo Elipsa. But is this the most recommended and the best option for reMarkable2 as well? I have ordered it and I am expecting to get it next Friday.

Thank you very much!
